app/scripts/modules/core/src/pipeline/config/stages/bakeManifest/helm/BakeHelmConfigForm.tsx

```diff
@@ -137,6 +137,14 @@ export class BakeHelmConfigForm extends React.Component<IFormikStageConfigInject
           pipeline={this.props.pipeline}
           stage={stage}
         />
+        <StageConfigField label=" File" helpKey="pipeline.config.bake.manifest.helm.chartFilePath">
```

Comment:
I could use a hand making this field appear only when the selected artifact is of type `git/repo` aka ArtifactTypePatterns.GIT_REPO.